packer/HWIMO-BUILD

```diff
@@ -4,7 +4,13 @@
 # input (enviromental variable):
 # * OS_VER        -   ubuntu-14.04 or ubuntu-16.04
 # * BUILD_TYPE    -   vmware or virtualbox
-# * RACKHD_INSTALL_ANSIBLE - which is the ansible yml playbook used. now are rackhd_package or rackhd_local
+# * RACKHD_INSTALL_ANSIBLE - which is the ansible yml playbook used. now are rackhd_ci_builds, rackhd_release or rackhd_local
+# * RACKHD_VERSION        - optional , by default is blank. used for rackhd_ci_builds.yml to install specific version of rackhd.deb, and moreover, this version used to version the vagrant box when uploading to ATLAS
```

Comment:
Nope... It's  not decided yeah..
but I added comments to refer to bintray.com/rackhd to find correct version..